extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/cors/CORSFilter.java

```diff
@@ -121,4 +119,13 @@ public void handle(RoutingContext event) {
             }
         }
     }
-}
+
+    private boolean containsOnlyWildcard(Optional<List<String>> optionalList) {
+        if (!optionalList.isPresent()) {
+            return true;
+        }
+
+        List<String> list = optionalList.get();
+        return list.isEmpty() || list.size() == 1 && "*".equals(list.get(0));
```

Comment:
```suggestion
        return list.isEmpty() || (list.size() == 1 && "*".equals(list.get(0)));
```
always better to be explicit.

Reply:
Yeah I think this is important too